app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+from app.errors import NotWearingMaskError
+from app.errors import OutdatedVaccineError
+from app.errors import NotVaccinatedError

```

Comment:
use inline import, it will increase readability
`from app.errors import` and comma separated
Also you can
```
from app.errors import (
   .....
) 
```